### PR TITLE
Agt2hc/feat/robfw 366/replace robotframework excellib

### PR DESCRIPTION
**Pull request by tranhoangnguyen193**
This branch do:
- ROBFW-366: replace robotframework-excellibrary-xwfintech(https://pypi.org/project/robotframework-excellibrary-xwfintech/) by robotframework-excellib](https://pypi.org/project/robotframework-excellib/
- ROBFW-347: support workspace for vscodium. Warning when change RobotTest folder location while install
- Change pypandoc package to pybandoc_binary package to fix test errors.

### CODE REVIEW

File: scripts/RobotFrameworkSetup.iss [hunk 1]
```diff
@@ -528,9 +529,18 @@ begin
...
   UsrDataDirPage.Values[0] := GetPreviousData('UsrDataDir',ExpandConstant('{sd}\RobotTest'));
+  PreviousUserDataDir := GetPreviousData('UsrDataDir',ExpandConstant(''));
 
 end;
 
+function NextButtonClick(CurPageID: Integer): Boolean;
+begin
+  Result := True;
+  if ((CurPageID = UsrDataDirPage.ID) and not (CompareText(ExpandConstant(''), PreviousUserDataDir) = 0)and not  	 (CompareText(UsrDataDirPage.Values[0], PreviousUserDataDir) = 0)) then
+    Result := Msgbox('You selected new folder for Workspace. Old working files need to be add to vscodium workspace manually.' + #13#10 +
```
Comment (test-fullautomation) [inline scripts/RobotFrameworkSetup.iss:540]: Hi Nguyen,

please write
"You selected a new folder for your workspace. Therefore your current files need to be added manually to your new VSCodium workspace."

Thank you,
 Thomas

**Review by test-fullautomation — APPROVED** [review]
Hi Nguyen,
please check my finding.
Thank you,
 Thomas

File: scripts/RobotFrameworkSetup.iss (hunk 1 above)
Comment (test-fullautomation) [reply scripts/RobotFrameworkSetup.iss:540]: uups. I approved accidentally. Please provide a new pull-request.

**Comment by tranhoangnguyen193** [comment]
Closed due to accidentally approve. Feedback and merge conflict will be fix on the new pull request.

**Pull request closed without merging**